components/usage/pkg/contentservice/usage_report.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package contentservice
+
+import (
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"time"
+)
+
+type InvalidSession struct {
+	Reason  string
+	Session db.WorkspaceInstanceForUsage
+}
+
+type UsageReport struct {
+	GenerationTime time.Time `json:"generation_time"`
```

Comment:
It's a shame that we have a mix of camel and snake case for the field names here. Is that something that we can change now?

Reply:
Yep, I think we still can

Reply:
https://github.com/gitpod-io/gitpod/pull/12511